modules/ROOT/pages/using-api-alerts.adoc

```diff
@@ -2,46 +2,42 @@
+* The response time of an API exceeds 60 seconds.
+* The number of API requests in 30 seconds is greater than 1000. 
 
-You can add one of the following types of alerts to an API:
+See xref:add-api-alert-task.adoc[Adding an API Alert].
 
-* Policy violation
-+
-An infraction of one or more policies that govern the API occurs.
-+
-* Request Count
-+
-Users request access to the API more times than allowed in a specified period of time.
-+
-* Response Code
+API Manager enables you to add the following API alerts to an API instance:
+
+* *Policy Violation*
 +
-The API returns one of these HTTP codes upon receiving a request: 400, 401, 403, 404, 408, 500, 502, or 503
+An infraction of one or more policies governing the API occurs.
+
+* *Request Count*
 +
-* Response Time
+Users request access to the API more times than allowed in a specified interval of time.
 +
-The response time of the API exceeds a specified period.
-
-A request count alert condition is similar to a policy alert that uses the rate limiting policy. Valid entries for the time interval option for a request count are in the range 1 - 999999999. These time intervals specified are implemented as sliding windows, not absolute minute boundaries, to prevent problems caused by data bursts.
+Specify time interval values within the range 1 - 999999999. To prevent data bursts, intervals are implemented as sliding windows instead of as absolute minute boundaries. See https://www.techopedia.com/definition/869/sliding-window[Sliding Windows].
 
-You create and configure an API alert by adding alerts to an API instance in API Manager.
-
-== Alert Notification
+* *Response Code*
++
+The API returns one of these HTTP codes upon receiving a request: 400, 401, 403, 404, 408, 500, 502, or 503.
 
-API Manager fires alerts when condition state changed from good to bad or from bad to good. No new alert fires if the condition remains in the same state. For a newly created alert initial state is undefined, so the very first occurrence of the bad state can be missed.
+* *Response Time*
++
+The response time of the API exceeds a specified timeout period. 
 
-When the alert fires, API Manager sends email notifications to you, and to others if configured.
+API alerts are triggered only when states change. though the first occurrence of a state change may be undetected due to newly created alerts not having defined states.
 
-When you set an alert, you select users in your Business Group who can receive email notifications about the alert. The notification includes information about events that trigger the alert as specified by the level of severity. Users receive two emails:
+When configuring an API alert, you select Business Group users who will receive email notifications about the alert. Notifications include information about events that trigger alerts as specified by severity level. Users receive the following:
```

Comment:
Just to avoid future tense...:
```suggestion
When configuring an API alert, you select Business Group users to receive email notifications about the alert. Notifications include information about events that trigger alerts as specified by severity level. Users receive the following:
```

Reply:
Fixed